client/my-sites/theme/main.jsx

```diff
@@ -212,12 +212,14 @@ class ThemeSheet extends React.Component {
 		return null;
 	}
 
-	previewAction = ( event ) => {
+	previewAction = ( event, type ) => {
 		if ( event.altKey || event.ctrlKey || event.metaKey || event.shiftKey ) {
 			return;
 		}
 		event.preventDefault();
 
+		this.props.recordTracksEvent( `calypso_theme_live_demo_${ type }_preview` );
```

Comment:
Rather than giving each event different names, I would use the same event (`calypso_theme_live_demo_preview_click`) with prop types of `screenshot` or `link`. (I'm not even sure we need to drill down that much, but it gives us the option).

Reply:
Sure thing, done! :)